apps/web/pages/event-types/index.tsx

```diff
@@ -741,9 +742,7 @@ const EventTypeListHeading = ({
         )}
         {profile?.slug && (
           <Link href={`${CAL_URL}/${profile.slug}`} className="text-subtle block text-xs">
-            {orgBranding
-              ? `${orgBranding.fullDomain.replace("https://", "").replace("http://", "")}${profile.slug}`
-              : `${CAL_URL?.replace("https://", "").replace("http://", "")}/${profile.slug}`}
+            {`${bookerUrl.replace("https://", "").replace("http://", "")}/${profile.slug}`}
```

Comment:
Because fullDomain doesn't have an ending slash now which is consistent with WEBAPP and CAL_URL. We can simplify the logic using bookerUrl